README.md updated (#14)
Grtzc/grtzc-app-react: README.md

# Global Realtime Timezone Converter (GRTZC)

This project aims to simplify converting between different timezones. The idea of this project comes from difficulties of using common timezone converter websites. I tried to use them for example converting time of CET to IRST and vice versa, but they seemed to be very complicated. So I decided to create a very simple but yet powerfull application to convert a timezone to many other timezones in realtime without querying server and unnecessary extra information.

> ## How it is working?

Reviewer: It is better to use "How it works?" instead of "How it is working?"

> ## How it is working?

Author: ```suggestion
## How it works?
```

For converting timezone, simply enter desired time (or current time by default) and then choose target timezone or timezones to see converted time in those timezones instantly.
You can also selected origin time's timezone if you want to convert timezone of in other timzone of your current timezone.

## Version and Future

Application is currently in the most early version (0.1) with very limited capabilities and poor design.

> But there are a lot of ideas to add new features and make it beatiful and user friendly.

Reviewer: Replace this "But there are a lot of ideas to add new features and make it beatiful and user friendly."
with:
"But there are lots of ideas to for new features to make it more beautiful and user-friendly."

> But there are a lot of ideas to add new features and make it beatiful and user friendly.

Author: ```suggestion
But there are lots of ideas to for new features to make it more beautiful and user-friendly.
```

> Current roadmap is:

Reviewer: ... The roadmap is...

> Current roadmap is:

Author: ```suggestion
The roadmap is:
```

> - Make page more beautiful and redisign it.

Reviewer: Redesign the app screen

> - Make page more beautiful and redisign it.

Author: ```suggestion
- Redesign the app screen
```

> - Use better component for time picker, so all style and usability errors will be vanished.

Reviewer: ...so all the style and usability errors...

> - Use better component for time picker, so all style and usability errors will be vanished.

Author: ```suggestion
- Use a better component for the time picker, so all the style and usability errors will vanish.
```

> - Use better drop down component for choosing timezones, so anyone can search for each timezone instead of searching it

Reviewer: Search instead of search?
"so anyone can search for each timezone instead of searching it"

Author: ای
```suggestion
- Use better drop down component for choosing timezones, so anyone can search for each timezone instead of picking it manually
```
